src/core/public/core_app/status/components/status_section.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiPageContent, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { StatusTable } from './status_table';
+import { FormattedStatus, getHighestStatus } from '../lib';
+import { StatusBadge } from './status_badge';
+
+interface StatusSectionProps {
+  id: string;
+  title: string;
+  statuses: FormattedStatus[];
+}
+
+export const StatusSection: FC<StatusSectionProps> = ({ id, title, statuses }) => {
+  const highestStatus = useMemo(() => getHighestStatus(statuses), [statuses]);
+
+  return (
+    <EuiPageContent grow={false}>
+      <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="s">
+            <h2>{title}</h2>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <StatusBadge status={highestStatus} data-test-subj={`${id}SectionStatusBadge`} />
```

Comment:
I've added per section (table) status badge. It should help identifying if the overall status was caused only by plugins, or if even core status are impacted. Also, this is more consistent with the http code we're now returning from the API, as `503` is only returned if core services are not available.